pom.xml

```diff
@@ -32,6 +32,7 @@ We use these goals frequently to keep the dependencies and plugins up-to-date:
     <description>Vavr (formerly called Javaslang) is an object-functional language extension to Java 8+.</description>
     <modules>
         <module>vavr</module>
+        <module>vavr-benchmark</module>
```

Comment:
Does a Java 9 build require now this module? Just asking...
I ask myself if now a jar is released for vavr-benchmark, too (i.e. to maven central).
Is the benchmark profile then still needed?

Reply:
It's in a separate commit, because it's obviously not strictly related :)
It should be part of the project so that it is compile-time checked. I will disable the deployment, it's a valid concern.

Reply:
Checked it, since it has tests only, there's nothing to deploy.
The `benchmark` profile has a run config also, so it still has its purpose - though we can remove the duplicate `modules` part, thanks for the note! :)

Reply:
(I had to test this module for Java 9 compatibility also, that's why I modified this part also)

Reply:
The benchmarks should not be part of the main build, even if it is a minimal version tests.
If there are assertions/checks, they should be moved to the vavr core unit tests.
Benchmarks are about measuring. The only valid case to run them as part of the tests would be that we break the build if the performance decreases. But that is not possible.

Reply:
The benchmarks aren't run during the build. Not even the assertions. just the compilation.
They are just part of the project, to make sure any refactoring is propagated there also.

Reply:
I see, that's cool.